.github/workflows/scripts/config-diff-post-comment.js

```diff
@@ -0,0 +1,61 @@
+module.exports = async ({ github, context, core, configDiff }) => {
+    try {
+        // Do not create comment if there are no configuration changes
+        if (!configDiff) {
+          console.log("No changes detected. Skipping comment creation.");
```

Comment:
You could use [the fine-grained `core` logging methods](https://github.com/actions/toolkit/tree/main/packages/core#logging).